boards/arm64/xenvm/xenvm-qemu.dts

```diff
@@ -11,75 +11,9 @@
  * with zephyr requirements.
```

Comment:
This description is not correct anymore. Also copyright should be `2023`, it was not created in 2020